manifests/m/Microsoft/Sqlcmd/0.8.1/Microsoft.Sqlcmd.locale.en-US.yaml

```
@@ -0,0 +1,36 @@
+# Created with YamlCreate.ps1 v2.1.5 $debug=NVS1.CRLF.5-1-19041-1682.Win32NT
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.2.0.schema.json
+
+PackageIdentifier: Microsoft.Sqlcmd
+PackageVersion: 0.8.1
+PackageLocale: en-US
+Publisher: Microsoft Corporation
+PublisherUrl: https://github.com/microsoft/go-sqlcmd
+PublisherSupportUrl: https://github.com/microsoft/go-sqlcmd
+# PrivacyUrl: 
+# Author: 
+PackageName: Sqlcmd
```

```suggestion
PackageName: Sqlcmd Tools
```
This must match Control Panel.